### PR TITLE
Mention the user's price after buying/selling.

### PR DESCRIPTION
**Pull request by ghost**
As the title says, after buying/selling an item, after the :tada:, it will say "Now you have [blobcoin emoji here] <new price>."

### CODE REVIEW

File: commands/store.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Store extends Command {
             }
             await this.client.db.giveUserItem(connection, message.guild.id, message.author.id, storeItem.id, amount);
             await connection.query('COMMIT');
-            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:`);
+            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada: Now you have <:blobcoin:386630453224013824> ${userData.currency}.`);
```
Comment (OGNova) [inline commands/store.js]: I'd rather this be `return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:\nYou now have <:blobcoin:386630453224013824> ${userData.currency}.`);`

Same goes for line 97.

**Review by 7bitlyrus — DISMISSED** [review]

**Review by OGNova — DISMISSED** [review]

File: commands/store.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ class Store extends Command {
             }
             await this.client.db.giveUserItem(connection, message.guild.id, message.author.id, storeItem.id, amount);
             await connection.query('COMMIT');
-            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:`);
+            return message.channel.send(`${message.author} You have bought ${amount}x ${storeItem.name} :tada:\nYou now have <:blobcoin:386630453224013824> ${userData.currency}.`);
```
Comment (scarletcafe) [inline commands/store.js]: userData is not in this scope, but if it was, it would be out-of-date anyway.

File: commands/store.js [hunk 3]
```diff
@@ -94,7 +94,7 @@ class Store extends Command {
             }
             await this.client.db.giveUserCurrency(connection, message.guild.id, message.author.id, returnPrice * amount);
             await connection.query('COMMIT');
-            return message.channel.send(`${message.author} You have sold ${amount}x ${storeItem.name} :tada:`);
+            return message.channel.send(`${message.author} You have sold ${amount}x ${storeItem.name} :tada:\nYou now have <:blobcoin:386630453224013824> ${userData.currency}.`);
           } else
```
Comment (scarletcafe) [inline commands/store.js]: Ditto, userData is not in scope.

File: commands/store.js (hunk 2 above)
Comment (scarletcafe) [reply commands/store.js]: In this particular instance, `deducted` already contains the result from `takeUserCurrency`, which returns an anon object including the `currency` field for the user. You should use `${deducted.currency}` instead.

File: commands/store.js (hunk 3 above)
Comment (scarletcafe) [reply commands/store.js]: As for here, `deducted` contains the other side of the deal and isn't as useful to us. We can instead store the value from `giveUserCurrency`, which like `takeUserCurrency`, returns a `currency` field:
```js
const updatedUser = await this.client.db.giveUserCurrency(...);
...
`You now have ${updatedUser.currency}`
```
etc

**Review by scarletcafe — CHANGES_REQUESTED** [review]
Doesn't work in current state; see review.

**Review by OGNova — APPROVED** [review]

**Review by scarletcafe — APPROVED** [review]
Tested and works as intended now. Also passes CI locally, even though Circle doesn't want to build for non-contribs.
Personally I'd like the message to be a bit less standoffish in prose, but this is a minor issue.

**Pull request merged**